opentracing-api/src/main/java/io/opentracing/Span.java

```diff
@@ -45,4 +161,10 @@
      * @see Span#context()
      */
     void finish(long finishMicros);
+
+    /**
+     * Defined as equivalent to calling {@code tracer.scopeManager().activate(this)} on the {@link Tracer} instance
+     * that started this {@link Span} instance.
+     */
+    Scope activate();
```

Comment:
only meaningful change to the Span interface... this is just sugar and doesn't need to be here (but it does save a lot of typing and complexity).